CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,92 @@
-﻿
+﻿using System;
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int Addnums(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = i + sum;
+
+            }
+            if (i == 0)
+            {
+                return 0;
+            }
+
+            else
+            {
+                return i;
+            }
+        }
+
+
+        public int Subnums(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = (-sum) + (-i);
```

Comment:
This line is somewhat confusing. Could it be re-written to be a little more clear what's happening?